.github/workflows/tests.yml

```diff
@@ -0,0 +1,72 @@
+name: Unit tests
+
+on: [push]
+
+env:
+  RUST_BACKTRACE: full
+
+jobs:
+  unit:
+    name: "Unit Tests"
+    runs-on: ubuntu-latest
```

Comment:
is this what warfa has?

Reply:
This is just CI, not nightlies. We would need a considerably more complicated system for that.

Reply:
oh right sorry I was still thinking of nightlies in my head ever since we discussed it in the egraph channel.